### ISSUE TITLE
Question about corrective LM loss

### ISSUE THREAD

**Issue by elmadany** [lucidrains/coco-lm-pytorch#2]
Hi @lucidrains ,

Thanks for your great repo!

I looked at your code: coco_lm_pytorch.py. I see there are three losses in line 242.
weighted_loss = self.cl_weight * cl_loss + self.gen_weight * mlm_loss + self.disc_weight * disc_loss

cl_loss is the contrastive loss, mlm_loss is the loss of the auxiliary generator, and disc_loss is the loss of binary discrimination. 
I wonder where the LM loss of corrective language modeling loss is. Could you point me?

Best,
Abdul.
